File: Embassat/Categories/UIFont+EMAdditions.m
//
//  UIFont+EMAdditions.m
//  Embassat
//
//  Created by Joan Romano on 27/03/14.
//  Copyright (c) 2014 Crows And Dogs. All rights reserved.
//

#import "UIFont+EMAdditions.h"

static NSString *const ApercuMedium = @"Apercu-Medium";
static NSString *const NoeDisplayBold = @"NoeDisplay-Bold";

@implementation UIFont (EMAdditions)

+ (UIFont *)em_detailFontOfSize:(CGFloat)size
{
    return [UIFont fontWithName:ApercuMedium size:size];
}

+ (UIFont *)em_titleFontOfSize:(CGFloat)size
{
    return [UIFont fontWithName:NoeDisplayBold size:size];
}

@end

File: Embassat/Categories/UIFont+EMAdditions.swift
//
//  UIFont+EMAdditions.swift
//  Embassa't
//
//  Created by Joan Romano on 9/24/15.
//  Copyright © 2015 Crows And Dogs. All rights reserved.
//

import Foundation

extension UIFont {    
    class func detailFont(ofSize size: Float) -> UIFont? {
        return UIFont(name: "Apercu-Medium", size: CGFloat(size))
    }
    
    class func titleFont(ofSize size: Float) -> UIFont? {
        return UIFont(name: "NoeDisplay-Bold", size: CGFloat(size))
    }
}
